Headlines/src/Controllers/NewsTableViewController.swift

```diff
@@ -10,11 +10,7 @@ import UIKit
 import SafariServices
 import ViewAnimator
 
-class NewsTableViewController: UITableViewController,
-                                NewsCellViewModelDelegate,
-                                UIViewControllerTransitioningDelegate,
-                                SFSafariViewControllerDelegate,
-                                TabbedViewController {
+class NewsTableViewController: UIViewController {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Canillitapp/headlines-iOS/blob/962ff671bf260eb1c01ac4072725c47d0275e505/Headlines/src/Controllers/NewsTableViewController.swift#L13"></span>Type body should span 200 lines or less excluding comments and whitespace: currently spans 244 lines</td>
    </tr>
  </tbody>
</table>